DOKS-wordpress/README.md

```diff
@@ -761,25 +763,30 @@ Now, open a web browser and navigate to [localhost:9150/metrics](http://127.0.0.
-Plugins are the building blocks of your WordPress site. They bring in important functions to your website, whether you need to add contact forms, improve SEO, increase site speed, create an online store, or offer email opt-ins. Whatever you need your website to do can be done with a plugin.
-
-Below you can find a list of recommended plugins:
+### Configuring the NitroPack Plugin
 
-- [Contact Form by WPForms](https://wordpress.org/plugins/wpforms-lite/): allows you to create beautiful contact forms, feedback form, subscription forms, payment forms, and other types of forms for your site.
+A CDN is required to speed up the delivery of content while taking the load off of your web server and minimizing latency while hosting your WordPress static content such as images, CSS, JavaScript and video streams. You will next configure the [Nitropack](https://wordpress.org/plugins/nitropack/) plugin in your Wordpress application.
 
-- [MonsterInsights](https://wordpress.org/plugins/google-analytics-for-wordpress/): is the best Google Analytics plugin for WordPress. It allows you to “properly” connect your website with Google Analytics, so you can see exactly how people find and use your website.
+**Note:**
+The wordpress admin password configured in the `values.yaml` of the Wordpress helm chart fails when attempting to log into WordPress admin. To change the password you need to connect to the the database and reset it. Follow this [article](https://docs.digitalocean.com/products/databases/mysql/how-to/connect/) on how to connect to a MySQL Database Cluster. Follow this [article](https://wordpress.org/support/article/resetting-your-password/) on how to change the password.
 
-- [All in One SEO](https://wordpress.org/plugins/all-in-one-seo-pack/): helps you get more visitors from search engines to your website. While WordPress is SEO friendly out of the box, there is so much more you can do to increase your website traffic using SEO best practices.
+Next you will configure the NitroPack plugin:
 
-- [SeedProd](https://wordpress.org/plugins/coming-soon/): is the best drag and drop page builder for WordPress. It allows you to easily customize your website design and create custom page layouts without writing any code.
+1. Navigate to the admin section of your WordPress instalation by going to: <https://YOUR_WORDPRESS_DOMAIN_HERE/wp-admin> and login with your wordpress admin credentials.
+2. Click on the `Plugins` menu item and `Add New` sub-menu.
+3. Search for the `Nitropack` plugin and from the results page click on the `Install Now` button. After the installation is complete, click on the `Activate` button. You should now see the plugin added added to your list of plugins
+4. Click on the `Settings` link under the plugin name. From this page click on the `Connect to NitroPack` button. This will let you log in or create an account with NitroPack.
+5. The NitroPack.io dashboard page should be opened with information related to the plan, optimized pages etc.
```

Comment:
I would use maybe a link here, rather than `NitroPack.io`.

Reply:
Will fix this in the next commit